src/common/global-hot-key/global-hot-key-helpers.ts

```diff
@@ -3,6 +3,7 @@ import { GlobalHotKeyKey } from "./global-hot-key-key";
 import { GlobalHotKeyModifier } from "./global-hot-key-modifier";
 
 export function isValidHotKey(hotKey: GlobalHotKey) {
-    return Object.keys(GlobalHotKeyKey).find((k) => k === hotKey.key) !== undefined
-        && (hotKey.modifier ? Object.keys(GlobalHotKeyModifier).find((m) => m === hotKey.modifier) : true);
+    return Object.values(GlobalHotKeyModifier).includes(hotKey.modifier!) &&
+        Object.values(GlobalHotKeyModifier).includes(hotKey.secondModifier!) &&
+        Object.values(GlobalHotKeyKey).includes(hotKey.key || "");
```

Comment:
Why is the `|| ""` needed?

Reply:
@tvollstaedt ?

Reply:
I don't remember anymore why I did this. `key` is not optional, so it should be always there.

```suggestion
        Object.values(GlobalHotKeyKey).includes(hotKey.key);
```